### ISSUE TITLE
Where is version 0.24+1?

### ISSUE THREAD

**Issue by asbjornu** [libgit2/libgit2sharp#1492]
Where can I find version `0.24+1` that is written about [here](https://github.com/libgit2/libgit2sharp/blob/master/CHANGES.md#v024--1)? I believe I'm seeing the problems (in GitTools/GitVersion#1248) that the following might fix:

> The exception thrown when the native library cannot be loaded is now able to be caught and will no longer crash the process.

But I can't find any newer versions of LibGit2Sharp than version `0.24.0` anywhere.

**Comment by ethomson** [comment]
There's no released version that includes this yet - `0.24+1` implies the next version.

This is available in prerelease nuget packages, though.

**Comment by ethomson** [comment]
Yeah, it's a bit confusing, I admit.  Probably something that I shoudld fix.

In any case, 0.25.0-preview-xxxx supports .NET Standard and .NET Framework, so it should be broadly compatible.  Does that not work for you?  That particular change could be backported without too much trouble, probably.

**Comment by asbjornu** [comment]
I haven't attempted an upgrade to `0.25.0` yet. If a backport to a `0.24.1` is possible, I would highly appreciate it. 🙏

**Comment by asbjornu** [comment]
I've now upgraded to `v0.25-preview-0033` in GitVersion (GitTools/GitVersion#1248), but as can seen [here](https://ci.appveyor.com/project/GitTools/gitversion/build/4.0.0-PullRequest1248.1382.build.1403), the build fails with `ExitCode: 1` there as well. 😕

**Comment by ethomson** [comment]
@asbjornu How can I help?

**Comment by asbjornu** [comment]
@ethomson: If you could help upgrading libgit2sharp in GitVersion in a PR that doesn't end up breaking the build, that would be extremely helpful.
